Adds 45 min, 90 min and 2 hour before alarms options to calendar events.
Index: /usr/palm/applications/com.palm.app.calendar/app/controllers/edit-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.calendar/app/controllers/edit-assistant.js
+++ /usr/palm/applications/com.palm.app.calendar/app/controllers/edit-assistant.js
@@ -449,7 +449,10 @@ var EditAssistant = Class.create({
 						 '-PT10M', //Do not localize
 						 '-PT15M', // Do NOT Localize
 						 '-PT30M', // Do NOT Localize
+						 '-PT45M',      // Customized: Add 45 minute option
 						 '-PT1H', // Do NOT Localize
+						 '-PT90M',      // Customized: Add 90 minute option
+						 '-PT2H',       // Customized: Add 2 hour option
 						 '-P1D' // Do NOT Localize
 						]
 						},
Index: /usr/palm/applications/com.palm.app.calendar/app/controllers/prefs-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.calendar/app/controllers/prefs-assistant.js
+++ /usr/palm/applications/com.palm.app.calendar/app/controllers/prefs-assistant.js
@@ -3,7 +3,7 @@
 var THIRTY_MINUTES = $L("30 minutes");
 var ONE_HOUR = $L("1 hour");
 var TWO_HOURS = $L("2 hours");
-var numTimedAlarm = 7;		
+var numTimedAlarm = 10;   // Customized: Changed 7 to 10. See corresponding change in BuildAlarmsMenu
 var PrefsAssistant = Class.create({
 				 
 	initialize: function() {
@@ -219,7 +219,10 @@ var PrefsAssistant = Class.create({
 									this.alarms[4].command,
 									this.alarms[5].command,
 									this.alarms[6].command,
-				
+									this.alarms[7].command,  // Customized: Added 3 new options, so need
+									this.alarms[8].command,  // Customized: to account for 10 items in
+									this.alarms[9].command,  // Customized: the list instead of 7.
+		
 									
 									this.allDayAlarms[1].command,
 									this.allDayAlarms[2].command,
@@ -638,7 +641,10 @@ var PrefsAssistant = Class.create({
 		{label: '', command: '-PT10M'},		//Do Not localize
 		{label: '', command: '-PT15M'},		// Do NOT Localize
 		{label: '', command: '-PT30M'},		// Do NOT Localize
+		{label: '', command: '-PT45M'},          // Customized: Added 45 minute option
 		{label: '', command: '-PT1H'},		// Do NOT Localize
+		{label: '', command: '-PT90M'},          // Customized: Added 90 minute option
+		{label: '', command: '-PT2H'},           // Customized: Added 2 hour option
 		{label: '', command: '-P1D'}			// Do NOT Localize	
 	],
 	
